README.md

```diff
@@ -98,26 +99,32 @@ The event name and what's in the `data` for each event handler:
   * `event` The raw event
   * `senderId` The ID of the sender
   * `session` [A Session object](#the-session-object) you can mutate
-  * `text` Message content, `event.message.text` for text events
+  * `payload` Message content, `event.message.text` for text events
```

Comment:
> Making text ambiguous isn't a good option

Agreed.

Moving to `text` and `normalizedText` would change the behavior of the `text` (`source`: `text`) event, which is likely the most commonly used event and thus, would be a higher impact. That said, `text` and `normalizedText` are easier to understand from both an SDK contributor and application developer perspective. Since we are at an early stage still, I think the breaking change no `text` is worth getting this right in an understandable way, so I'm on board w/`text` and `normalizedText`.